ccmlib/node.py

```diff
@@ -812,6 +812,13 @@ def run_sstablemetadata(self, output_file=None, datafiles=None, keyspace=None, c
         if output_file == None:
             return results
 
+    def get_sstables(self, output_file=None, datafiles=None, keyspace=None, tables=None):
+        cdir = self.get_install_dir()
+        sstablemetadata = common.join_bin(cdir, os.path.join('tools', 'bin'), 'sstablemetadata')
```

Comment:
I don't understand what's going on here. You never call `sstablemetadata`, or forward this variable